src/job/jobHandler.ts

```diff
@@ -233,15 +233,13 @@ export abstract class JobHandler {
         if (typeof pathPrefix !== 'undefined' && pathPrefix !== null) {
             envVars += `PATH_PREFIX=${pathPrefix}\n`
         }
-        // const snootyFrontEndVars = {
-        //   'GATSBY_FEATURE_FLAG_CONSISTENT_NAVIGATION': this._config.get<boolean>("gatsbyConsitentNavFlag"),
-        //   'GATSBY_FEATURE_FLAG_SDK_VERSION_DROPDOWN': this._config.get<boolean>("gatsbySDKVersionDropdownFlag"),
+        const snootyFrontEndVars = {
+          'GATSBY_BASE_URL': this._config.get<String>("gatsbyBaseUrl"),
```

Comment:
Asking as a reminder: the purpose of this step is to have the autobuilder deliver the value of this variable set in parameter store to the frontend's env file, right?